jbpm-system-test/jbpm-timer-migration-test/src/test/resources/etc/Dockerfile

```
@@ -0,0 +1,49 @@
+# Multi-stage Dockerfile
+
+# First, build kjar from a maven image
+
+ARG IMAGE_NAME
+ARG NO_PATCH
+
+FROM maven:3.6.3-openjdk-8-slim as builder
+ARG NO_PATCH
+LABEL autodelete="true"
+COPY etc/kjars/ /etc/kjars/
+RUN mvn --file /etc/kjars/timer-sample/pom.xml --batch-mode install -DskipTests
+
+#########################################################
+# Dockerfile that provides the image for KIE Server
+#########################################################
+
+FROM $IMAGE_NAME
+ARG NO_PATCH
+
+RUN echo "Building from $IMAGE_NAME"
+
+COPY --from=builder /root/.m2 /opt/jboss/.m2
+
+ENV KIE_SERVER_PROFILE standalone
+
+COPY etc/drivers /etc/drivers/
+ADD etc/jbpm-custom.cli $JBOSS_HOME/bin/jbpm-custom.cli
+
+ADD etc/jbpm-flow-7.69.0.Final.jar /WEB-INF/lib/jbpm-flow-7.69.0.Final.jar
+
+RUN if [ -z "$NO_PATCH" ]; then \
+rm $JBOSS_HOME/standalone/deployments/kie-server.war/WEB-INF/lib/jbpm-flow-7.69.0.Final.jar; \
```

There is an issue here. Since the image we are using it's based on `quay.io/kiegroup/kie-server-showcase:7.70.0.Final` the `WEB-INF/lib/jbpm-flow-7.69.0.Final.jar` will not exist.

Must be

```suggestion
rm $JBOSS_HOME/standalone/deployments/kie-server.war/WEB-INF/lib/jbpm-flow-7.70.0.Final.jar; \
```

Though if this is the case, maybe the patch is already included in 7.70.0.Final version?